### ISSUE TITLE
We need a logo

### ISSUE THREAD

**Issue by Jaykul** [PoshCode/PSGit#29]
What do you think of this?
![psgit](https://cloud.githubusercontent.com/assets/192942/7665059/ca092fec-fb74-11e4-9aaa-36a4eee14001.png)

**Comment by jrich523** [comment]
The text should be level, its just slightly off and it looks weird.. also, having the git orange in there would give it a little bit of life

![psgit_logo](https://cloud.githubusercontent.com/assets/2077508/7666683/c03432ae-fba2-11e4-9144-38dd75b52cab.png)

**Comment by KirkMunro** [comment]
I like the first one better.  And if you want to be true to git, use the
Adelle font (which is apparently the font that the git logo was designed
with).

## 

*Kirk Munro *
Poshoholic, Microsoft MVP
Blog http://poshoholic.com/ | Twitter http://twitter.com/poshoholic |
LinkedIn http://ca.linkedin.com/in/kirkmunro | GitHub
http://github.com/KirkMunro | Facebook
http://www.facebook.com/#%21/kirk.munro

Need a PowerShell SME for a project at work? Contact me
http://poshoholic.com/contact-me/!

**Comment by Jaykul** [comment]
OK, I found the Adelle font and lined them up....
![psgit](https://cloud.githubusercontent.com/assets/192942/7668143/9e08478e-fbf6-11e4-9697-87e570645475.png)

**Comment by Jaykul** [comment]
![psgit](https://cloud.githubusercontent.com/assets/192942/7668239/5fc9f6f8-fbfa-11e4-896e-e5918348635a.png)
Or we could go with the sunset theme ;-)

**Comment by Jaykul** [comment]
![psgit](https://cloud.githubusercontent.com/assets/192942/7668260/07352dbc-fbfc-11e4-8ca5-7764f97705b2.png)
What if I make the "git" font the same size as the "PS"

**Comment by Jaykul** [comment]
![psgit](https://cloud.githubusercontent.com/assets/192942/7668270/c8888414-fbfc-11e4-9a7f-76d3c58d7120.png)

Two tone icon? I'm reluctant to go full PowerShell on that icon and give it the "gel" look with gradients in three different directions and glowing white shadows ...

**Comment by Jaykul** [comment]
![psgit](https://cloud.githubusercontent.com/assets/192942/7668310/609fa51e-fbff-11e4-8977-55bc49148fa1.png)
OK, not _that_ reluctant, here's a more PoweShelly icon ;-)

**Comment by Jaykul** [comment]
![psgit](https://cloud.githubusercontent.com/assets/192942/7668993/d9641d9e-fc28-11e4-9d30-e8f3216029c4.png)
OK, not that reluctant, here's a more PoweShelly icon ;-)

**Comment by Jaykul** [comment]
![psgit](https://cloud.githubusercontent.com/assets/192942/7669016/6e2993fe-fc2a-11e4-8f82-be4c9a03df4a.png)
Or we could have all of the above!
Hmmm, maybe not. I don't think I like this one as much as any of the others.

**Comment by bobfrankly** [comment]
Here's my two cents, or images in their oversize-edness:
![poshgit](https://cloud.githubusercontent.com/assets/2328006/8091029/153df8ee-0f66-11e5-99e3-47a4f598a29c.png)
![poshgit2](https://cloud.githubusercontent.com/assets/2328006/8091031/169e98d8-0f66-11e5-8fc8-ea1935832b97.png)

**Comment by Jaykul** [comment]
That's a cute idea -- would it break if we tilted the whole thing to match the actual PowerShell logo?

**Comment by jrich523** [comment]
That looks good, what about making the >_ blue?

or even pushing the > inside (on top) of the git logo

**Comment by bobfrankly** [comment]
Asked and answered. If you have a specific blue in mind, hex me.

![poshgit5](https://cloud.githubusercontent.com/assets/2328006/8091744/9cf0414e-0f6a-11e5-803f-bba7c13250be.png)

![poshgit3](https://cloud.githubusercontent.com/assets/2328006/8091548/454e6200-0f69-11e5-818c-8a0b97423ddb.png)
![poshgit4](https://cloud.githubusercontent.com/assets/2328006/8091549/45508332-0f69-11e5-90b2-ca43309dd685.png)

The Blue inside makes me think of ordering pizza...

**Comment by jrich523** [comment]
338ce7

that should be it i think

**Comment by bobfrankly** [comment]
With 338CE7 Blue. Most of the powershell logos seem to have gradient forms of a darker and lighter blue. It's not going to meld perfiectly with the flat orange of the GIT logo.

![poshgit6](https://cloud.githubusercontent.com/assets/2328006/8092824/81213002-0f71-11e5-9d4a-a01ce8355f78.png)
![poshgit7](https://cloud.githubusercontent.com/assets/2328006/8092825/81231bc4-0f71-11e5-8feb-d6a54f6774ff.png)

**Comment by Jaykul** [comment]
I don't think that's the right shade at all @jrich523 -- but it doesn't really matter, because those colors are awful together :stuck_out_tongue:

I tried a version where I tilted the git logo to match the tilt on the PowerShell logo, but it doesn't look right either. Here's a disposable version
![gitps](https://cloud.githubusercontent.com/assets/192942/8092882/195231aa-0f8b-11e5-9199-37e3a8830ba9.png)

**Comment by bobfrankly** [comment]
Another log on the fire...
![poshgit8](https://cloud.githubusercontent.com/assets/2328006/8093088/999cd1b6-0f73-11e5-82e1-f5613f4cddd1.png)

**Comment by Jaykul** [comment]
Oh, how about if I make it sublter ...
![gitps](https://cloud.githubusercontent.com/assets/192942/8093194/801e2270-0f8d-11e5-8eb6-01d753649057.png)

**Comment by bobfrankly** [comment]
Messing with Paletton: http://paletton.com/palette.php?uid=3070u0kpcu7fvGmkExttCq3vCk-

![poshgit9](https://cloud.githubusercontent.com/assets/2328006/8094960/323826b8-0f7f-11e5-8b26-12cb02bae190.png)

**Comment by bobfrankly** [comment]
Last iteration I'll throw up for now.

![poshgit10](https://cloud.githubusercontent.com/assets/2328006/8095360/66d65068-0f82-11e5-949f-36f16f921977.png)

**Comment by Jaykul** [comment]
Maybe start from the PowerShell blue and force the orange to adapt instead?
http://paletton.com/palette.php?uid=33F0K0kWQgnSBuMUBo1XralY33j

**Comment by Jaykul** [comment]
![psgit3](https://cloud.githubusercontent.com/assets/192942/8096473/8cecd20e-0fa5-11e5-87f7-f51ce75b6dc9.png)

Might be better to change the colors to be more complementary, but I like this look :-)

**Comment by mwjcomputing** [comment]
This looks sweet.

On Jun 10, 2015, at 7:19 PM, Joel Bennett <notifications@github.com<mailto:notifications@github.com>> wrote:

[psgit3]https://cloud.githubusercontent.com/assets/192942/8096473/8cecd20e-0fa5-11e5-87f7-f51ce75b6dc9.png

##

**Comment by KirkMunro** [comment]
Why is there a white gap in the blue that is behind the Git portion of the
logo?

## 

*Kirk Munro *
Poshoholic, Microsoft MVP
Blog http://poshoholic.com/ | Twitter http://twitter.com/poshoholic |
LinkedIn http://ca.linkedin.com/in/kirkmunro | GitHub
http://github.com/KirkMunro | Facebook
http://www.facebook.com/#%21/kirk.munro

Need a PowerShell SME for a project at work? Contact me
http://poshoholic.com/contact-me/!

**Comment by Jaykul** [comment]
Hehe, I wondered if that would bother anyone.  It's because the blue is exactly the same shape as the orange.

**Comment by KirkMunro** [comment]
No, not the top left one..the other one in the middle (partly behind the
git "dot" on the right).

## 

*Kirk Munro *
Poshoholic, Microsoft MVP
Blog http://poshoholic.com/ | Twitter http://twitter.com/poshoholic |
LinkedIn http://ca.linkedin.com/in/kirkmunro | GitHub
http://github.com/KirkMunro | Facebook
http://www.facebook.com/#%21/kirk.munro

Need a PowerShell SME for a project at work? Contact me
http://poshoholic.com/contact-me/!

**Comment by bobfrankly** [comment]
@KirkMunro It's as he said, the blue is just a cloned copy of the orange, as there is a line of transparency in the orange, there is also on in the blue. Because of the offset of the blue, it shows in that "git dot" on the right.

@Jaykul  I saw it, but as we're just tossing ideas around, I wasn't going to make a fuss unless it was the final version.

**Comment by Jaykul** [comment]
I could make a version with the blue layer not having the git branch cutout ... 
or slide it over so the edge of the branch shows at the top of the orange ... 

I could also swap the blue for the "real" color of the ps host background ... #012456, but if we did that, I think we would need to change the orange to that [split-complement orange](http://paletton.com/palette.php?uid=33F0K0kWQgnM-puT8l4ZfaC%2BN3k) #821400 or at least darken it to #CB2000  

Edit: I thought that Github did color swatches like slack:
![paleton palette](http://api.paletton.com/img?size=25&uid=33F0K0kWQgnM-puT8l4ZfaC%2BN3k)

**Comment by KirkMunro** [comment]
Ok, I get it now. 😊

I like the logo btw, but the orange color feels a little too peachy to me.

-----Original Message-----

**Comment by bobfrankly** [comment]
of course, we're all arguing over shades of color on un-calibrated screens. If we're really going to do this, we should all have some S5X100s. I nominate Jaykul for funding :laughing:

**Comment by Jaykul** [comment]
Hey, I bought my ProArt monitor just for these occasions -- buy your own monitors :-P

**Comment by Jaykul** [comment]
![psgit31](https://cloud.githubusercontent.com/assets/192942/8132398/842b9d3a-10ef-11e5-8bdf-70437c8719bd.png)  Blue corrected ...

**Comment by Jaykul** [comment]
![psgit32](https://cloud.githubusercontent.com/assets/192942/8132478/09fdb894-10f0-11e5-9049-a9c894ed033e.png) split-complement orange (Ok, I think we can all agree that's a disaster)

**Comment by Jaykul** [comment]
![psgit33](https://cloud.githubusercontent.com/assets/192942/8132501/3134e0e0-10f0-11e5-8035-a83dcd420aed.png) That's #CB2000 ... 

I don't know about you guys, but the more I look at this, the more I like the lighter blue I was using before.

**Comment by Jaykul** [comment]
So yeah, how about this:

![psgit34](https://cloud.githubusercontent.com/assets/192942/8132735/d7b8cd0e-10f1-11e5-8352-7cadc5f15a50.png)

We can use just the icon part when we want that
![psgit35](https://cloud.githubusercontent.com/assets/192942/8132736/d9a7e3f2-10f1-11e5-8023-bbbc9fb1c6bc.png)

**Comment by jrich523** [comment]
i like this last one and the one you commented "blue corrected", i think i like the blue corrected one better, bit more bold and sharp, this one is a bit softer, both are good, just leaning more towards the bold

**Comment by KirkMunro** [comment]
I'm liking the bold as well.  The other feels too "faded" to be a logo.

## 

*Kirk Munro *
Poshoholic, Microsoft MVP
Blog http://poshoholic.com/ | Twitter http://twitter.com/poshoholic |
LinkedIn http://ca.linkedin.com/in/kirkmunro | GitHub
http://github.com/KirkMunro | Facebook
http://www.facebook.com/#%21/kirk.munro

Need a PowerShell SME for a project at work? Contact me
http://poshoholic.com/contact-me/!

**Comment by jrich523** [comment]
so i think we agree on that as a style, and just need to pick between the "blue corrected", which we can refer to as the bold, and then the above one, the soft...

so thats 1 for soft (jaykul) and 2 for bold 

**bob** the monitor is a valid point, with my 3 monitors, the soft one looks a tad more bold on one of them, however its the older more inaccurate one :)

**Comment by bobfrankly** [comment]
I'm in the "blue corrected" camp. It's not so much that it's "bolder", as it is that the blue and the orange appear to have matching saturation values (I'm a photoshop/photography nerd, also explains the monitor calibration knowledge :)

**Comment by Jaykul** [comment]
In [#CB2000](https://github.com/PoshCode/PSGit/issues/29#issuecomment-111516371) they have (almost) matching saturation (I thought it was exact, but I used an RGB approximation):
Blue: RGB: 1 36 86     HSL: 215° 98% 17%
Orange: RGB: 203 32 0   HSL: 9° 100% 40% 

In [Blue corrected](https://github.com/PoshCode/PSGit/issues/29#issuecomment-111515197) they're further off (same blue, less saturated, but lighter orange):

Blue: RGB: 1 36 86     HSL: 215° 98% 17%
Orange: RGB: 245 77 39  HSL: 11° 91% 56%

And of course, in the others, the blue much less saturated (and also slightly more red/green):
Blue: RGB: 40 80 121     HSL: 210° 50% 32%

**Comment by Jaykul** [comment]
![psgit36](https://cloud.githubusercontent.com/assets/192942/8140540/6f757fec-112a-11e5-9a68-d1ac30c01950.png)

So how about if I make the Orange match the saturation exactly?

Blue: #012456 RGB: 1 36 86 HSL: 215° 99% 34%
Orange: #f52f02 RGB: 245 47 2 HSL: 11° 99% 96%

I also cleaned up the text a little (recreated it from the font) and made the _ underbar go where the font would put it (although I rounded it, the Adelle underbar is rectangular).

**Comment by catthehacker** [comment]
Contrast is a bit off and it hurts. Imho it's time to drop serif fonts. How about using [microsoft/cascadia-code](https://github.com/tonsky/microsoft/cascadia-code) or [tonsky/FiraCode](https://github.com/tonsky/FiraCode)?

**Comment by scrthq** [comment]
@CatTheHacker - super thread resurrection, but it's using the same font as the Git logo itself... Anyway, Fira Code bare came into existence before this thread was last updated and Cascadia Code obviously didn't exist yet

**Comment by catthehacker** [comment]
> 
> 
> @CatTheHacker - super thread resurrection, but it's using the same font as the Git logo itself... Anyway, Fira Code bare came into existence before this thread was last updated and Cascadia Code obviously didn't exist yet

True, I didn't noticed the dates and I just jumped in to thread... Anyway, what is your point?

**Comment by scrthq** [comment]
@CatTheHacker My point is that the font chosen is purposeful (same or close to the same font as the existing Git logo), not preferential
